src/npm/components/MetApi.tsx

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable react/display-name */
+import React from 'react';
+import LineChart, { LineChartProps } from '../../components/charts/LineChart';
+import ThresholdChart, { ThresholdChartProps } from '../../components/charts/ThresholdChart';
+import { DataSourceID } from '../../utils/dataSourceMappings';
+import VisualiserWrapper from './VisualiserWrapper';
+
+export const MetApi = {
+    Line: ({
+        width = 600,
+        height = 300,
+        yLabel,
+        strokeColor,
+        colorBottom,
+    }: Omit<LineChartProps, 'data'>): JSX.Element => (
```

Comment:
Could the components be moved out of the object and typed with `React.FC<...>` instead? This would provide a more complete type and make it a bit easier to maintain.

Reply:
Makes sense, I will see what I can do.